src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -2046,5 +2046,219 @@ public C(string s)
     }
 }");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
+        [WorkItem(52385, "https://github.com/dotnet/roslyn/issues/52385")]
+        public async Task SingleLineStatement_NullCheck_BracesNone_SameLineFalse()
+        {
+            await new VerifyCS.Test
+            {
+                TestCode = @"
+using System;
+
+class C
+{
+    public C([||]object o)
+    {
+    }
+}",
+                FixedCode = @"
+using System;
+
+class C
+{
+    public C(object o)
+    {
+        if (o is null)
+            throw new ArgumentNullException(nameof(o));
+    }
+}",
+                Options =
+                {
+                    { CSharpCodeStyleOptions.PreferThrowExpression, false, NotificationOption2.Silent },
+                    { CSharpCodeStyleOptions.PreferBraces, PreferBracesPreference.None, NotificationOption2.Silent },
+                    { CSharpCodeStyleOptions.AllowEmbeddedStatementsOnSameLine, false, NotificationOption2.Silent },
```

Comment:
We can simplify this by just using the default severity.

```suggestion
                    { CSharpCodeStyleOptions.PreferThrowExpression, false },
                    { CSharpCodeStyleOptions.PreferBraces, PreferBracesPreference.None },
                    { CSharpCodeStyleOptions.AllowEmbeddedStatementsOnSameLine, false },
```